python/orca/src/bigdl/orca/learn/pytorch/estimator.py

```diff
@@ -25,58 +25,61 @@ def from_torch(*,
                    metrics=None,
-                   scheduler_creator=None,
-                   training_operator_cls=TrainingOperator,
-                   initialization_hook=None,
+                   backend="spark",
                    config=None,
+                   workers_per_node=1,
+                   scheduler_creator=None,
                    scheduler_step_freq="batch",
                    use_tqdm=False,
-                   workers_per_node=1,
                    model_dir=None,
-                   backend="spark",
                    sync_stats=False,
                    log_level=logging.INFO,
                    log_to_driver=True,
+                   training_operator_cls=TrainingOperator,
                    ):
         """
-        Create an Estimator for torch.
+        Create an Estimator for PyTorch.
 
-        :param model: PyTorch model or model creator function if backend="bigdl", PyTorch
-               model creator function if backend="horovod" or "ray"
-        :param optimizer: Orca/PyTorch optimizer or optimizer creator function if backend="bigdl"
-               , PyTorch optimizer creator function if backend="horovod" or "ray"
-        :param loss: PyTorch loss or loss creator function if backend="bigdl", PyTorch loss creator
-               function if backend="horovod" or "ray"
-        :param metrics: Orca validation methods for evaluate.
-        :param scheduler_creator: parameter for `horovod` and `ray` backends. a
-               learning rate scheduler wrapping the optimizer. You will need to set
-               ``scheduler_step_freq="epoch"`` for the scheduler to be incremented correctly.
-        :param config: parameter config dict, CfgNode or any class that plays a role of
-               configuration to create model, optimizer loss and data.
-        :param scheduler_step_freq: parameter for `horovod` and `ray` backends.
-               "batch", "epoch" or None. This will determine when ``scheduler.step`` is called. If
-               "batch", ``step`` will be called after every optimizer step. If "epoch", ``step``
-               will be called after one pass of the DataLoader. If a scheduler is passed in, this
-               value is expected to not be None.
-        :param use_tqdm: parameter for `horovod` and `ray` backends. You can monitor
-               training progress if use_tqdm=True.
-        :param workers_per_node: parameter for `horovod` and `ray` backends. worker
-               number on each node. default: 1.
-        :param model_dir: parameter for `bigdl` and `spark` backend. The path to save model. During
-               the training, if checkpoint_trigger is defined and triggered, the model will be
-               saved to model_dir.
-        :param backend: You can choose "horovod",  "ray", "bigdl" or "spark" as
-               backend. Default: `spark`.
+        :param model: A model creator function that takes the parameter "config" and returns a
```

Comment:
Is it necessary to keep `bigdl` backend related comments? such as PyTorch model if backend="bigdl"

Reply:
It will be deprecated in the future and not recommended and thus removed from the doc.